winterfell/test/testClaimService.js

```diff
@@ -5,26 +5,30 @@ var Claim = require('../models/claim');
 var ClaimService = require('../services/claimService');
 var User = require('../models/user');
 var UserService = require('../services/userService');
+var Form = require('../models/form');
 
 describe('ClaimService', function() {
-  var targetUser = undefined;
+  var server, targetUser;
 
-  before(function(done) {
-     // Remove all users and create one user
-     User.remove({}, function() {
-       var user = {
-         firstname: 'User1',
-         lastname: 'McUser',
-         email: 'user1@email.com',
-         password: 'testword',
-         state: User.State.ACTIVE
-       };
-       User.create(user, function(err, user) {
-         targetUser = user;
-         done();
-       });
-     });
-   });
+  before(function (done) {
+    this.timeout(10000);
```

Comment:
Oh I dont know why I set this, the test certainly doesnt take 10 seconds, will remove